### PR TITLE
Outliers

### PR DESCRIPTION
**Pull request by KFedak**
taking a break so pulling my branch changes to master. Will pick up later today or tomorrow by first pushing from master to my branch again.

### CODE REVIEW

**Comment by kbilsback** [comment]
:octocat:

**Comment by kbilsback** [comment]
yea! emojis 🌵 😸 :taco:

File: r_markdown/benzene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: both samples and fuel_burnt have a variable called "fuel" which is causing problems when you join (creates a variable fuel.x and then issues with your code does the line). I've fixed in commit...

File: r_markdown/benzene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: I'll apply the new style template to this in a new branch after we've merged...

File: r_markdown/process_metadata.Rmd
Comment (ngood13) [inline r_markdown/process_metadata.Rmd:96]: looks good

File: r_markdown/toluene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: Benzene or Toluene?

File: r_markdown/toluene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: Benzene or Toluene?

**Review by kbilsback — CHANGES_REQUESTED** [review]
Wait, how did those benzene and toluene files sneak in there? #roguemerge 😸

File: r_markdown/toluene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: drop x axis labels perhaps?

File: r_markdown/toluene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: I'll add some ## when I do the style update for ease of navigation...

File: r_markdown/benzene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: see comments on toluene that are also relevant to these (and visa versa...)

**Comment by KFedak** [comment]
waaaaa I have no idea! :crying_cat_face:

File: r_markdown/benzene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/benzene_emissionsfactor_qa.Rmd]: we should define a manual fuel color-scale so they always appear the same regardless of subsetting (Kelsey? 👍 )

File: r_markdown/toluene_emissionsfactor_qa.Rmd
Comment (ngood13) [inline r_markdown/toluene_emissionsfactor_qa.Rmd]: same as benzene (fuel conflict, fixed)

**Review by ngood13 — APPROVED** [review]
just make sure my commits are part of the merge to the master...

File: r_markdown/process_metadata.Rmd
Comment (kbilsback) [inline r_markdown/process_metadata.Rmd:173]: Nice! Comments!

**Comment by KFedak** [comment]
I finished making my comments in the "process_metadata.Rmd" file, and am now trying to merge back to the master.... but it looks like merging is blocked until a requested change is addressed, and I am blocked from making that change?

**Comment by ngood13** [comment]
Looks @kbilsback has requested changes, not sure who they are, they sound like trouble :-)

Also, I fixed some stuff in the voc markdowns that @rowdyspecialkbnewlyreformedenforcerofrules deleted, so you might want to look back and find them if needed...

**Review by kbilsback — APPROVED** [review]

**Comment by kbilsback** [comment]
@KFedak already fixed the changes! :octocat: 👍

**Pull request merged**
